django/db/migrations/state.py

```diff
@@ -403,6 +410,12 @@ def from_model(cls, model, exclude_rels=False):
                 elif name == "index_together":
                     it = model._meta.original_attrs["index_together"]
                     options[name] = set(normalize_together(it))
+                elif name == "indexes":
+                    indexes = model._meta.original_attrs["indexes"]
+                    for index in indexes:
+                        # Set index._name of all indexes by calling index.name
+                        index.name
```

Comment:
Because the `Index` class doesn't has it's `model` attribute set in `__init__` which is required in `get_name`.